nsswitch.conf

```diff
@@ -0,0 +1 @@
+hosts: files dns
```

Comment:
since we are getting more of these (nginx.conf too), we could make a conf dir to clean up top-level dir